.github/workflows/cypress.yaml

```diff
@@ -42,6 +42,7 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v2
         with:
+          env: UBUNTU_USERNAME=peter.makowski+100@canonical.com,UBUNTU_PASSWORD=jibfdq5hmq
```

Comment:
Didn't manage to get github secrets being passed correctly with `cypress-io/github-action@v2`, writing this inline for now.

If there's another test user that we should use please let me know.

Reply:
I'm not too sure about this @anthonydillon might know more

Reply:
Is this supposed to be a secret?

Reply:
would be nice if it was, but doesn't really have to be, it's a dummy test user